pkg/storaged/utils.js

```diff
@@ -236,49 +244,69 @@ export function get_block_link_parts(client, path) {
     if (!block)
         return;
 
-    let location, link;
+    let location, link, name, subname;
     if (client.mdraids[block.MDRaid]) {
         location = ["mdraid", client.mdraids[block.MDRaid].UUID];
         link = cockpit.format(_("RAID device $0"), mdraid_name(client.mdraids[block.MDRaid]));
     } else if (client.blocks_lvm2[path] &&
                client.lvols[client.blocks_lvm2[path].LogicalVolume] &&
                client.vgroups[client.lvols[client.blocks_lvm2[path].LogicalVolume].VolumeGroup]) {
-        const target = client.vgroups[client.lvols[client.blocks_lvm2[path].LogicalVolume].VolumeGroup].Name;
+        const lvol = client.lvols[client.blocks_lvm2[path].LogicalVolume];
+        const target = client.vgroups[lvol.VolumeGroup].Name;
         location = ["vg", target];
         link = cockpit.format(_("LVM2 volume group $0"), target);
+        name = target;
+        subname = lvol.Name;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18967-20230629-132526-4d4c19dd-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)